app/templates/dataset.html

```diff
@@ -58,8 +58,40 @@ <h2>CONP Portal | {{ data.title }} </h2>
         href="{{ href }}">{{ d.name }}</a></span>
     {% endfor %}</p>
   {% endif %}
+
+  {% if metadata.primaryPublications is defined and metadata.primaryPublications is not none %}
+  {% if metadata.primaryPublications|length > 1 %}
```

Comment:
this if is for datasets with more than one publication attached, they will be displayed as lists (CHBM will be one of those datasets but it is not yet part of conp-dataset)